plugins/catalog-import/src/components/StepPrepareCreatePullRequest/StepPrepareCreatePullRequest.tsx

```diff
@@ -102,10 +102,15 @@ export const StepPrepareCreatePullRequest = ({
   const catalogApi = useApi(catalogApiRef);
   const catalogImportApi = useApi(catalogImportApiRef);
   const errorApi = useApi(errorApiRef);
+  const configApi = useApi(configApiRef);
 
   const [submitted, setSubmitted] = useState(false);
   const [error, setError] = useState<string>();
 
+  const catalogFilename =
```

Comment:
This is repeated enough times that it's kinda worth making a local `useCatalogFilename()` hook 😅, or a `getCatalogFilename(config)` helper

Reply:
Good point, introduced both of them (and additional `getBranchName`) as these need to be accessed both in React components and in `CatalogImportApi`.